src/AddressableInterface.php

```diff
@@ -12,4 +12,6 @@ interface AddressableInterface
     public function address(): AddressInterface;
 
     public function subnet(): SubnetInterface;
+
+    public function contains(AddressableInterface $address): bool;
```

Comment:
Added so that you can check against a Address or network without knowing

Reply:
I don't see anything wrong with it. But I don't understand why you would need an Address::contains would be usefull. But for consistency I can get behind it.
Do you have an use-case: When do have an AddressableInterface and not an instance  of Network?